### PR TITLE
Update view-reports-for-atp.md

### PR DESCRIPTION
**Pull request by faoquong**
@chrisda

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/security/office-365-security/view-reports-for-atp.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ To view this report, in the [Security &amp; Compliance Center](https://protectio
 
 ![ATP File Types report](../../media/6e3f5d33-79aa-4b2d-938c-6ef135d9e54c.png)
   
-When you hover over a particular day, you can see the breakdown of types of malicious files that were detected by [ATP Safe Attachments](atp-safe-attachments.md) and [anti-spam &amp; anti-malware protection](anti-spam-and-anti-malware-protection.md).
+When you hover over a particular day, you can see the breakdown of types of malicious files that were detected by [ATP Safe Attachments](atp-safe-attachments.md) and [anti-spam &amp; anti-malware protection](anti-spam-and-anti-malware-protection.md). the aggregate view of the report only allows for 90 days filtering, while the detial view only allows for 10 days filtering.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/view-reports-for-atp.md]:
```suggestion
When you hover over a particular day, you can see the breakdown of types of malicious files that were detected by [ATP Safe Attachments](atp-safe-attachments.md) and [anti-spam &amp; anti-malware protection](anti-spam-and-anti-malware-protection.md). The aggregate view of the report allows for 90 days of filtering, while the detail view only allows for ten days of filtering. 
```

File: microsoft-365/security/office-365-security/view-reports-for-atp.md [hunk 2]
```diff
@@ -120,7 +120,7 @@ To view this report, in the [Security &amp; Compliance Center](https://protectio
 
 When you hover over a bar in the chart, you can see what actions were taken for detected email for that day.
 
-![ATP Message Disposition Report data for a day](../../media/68d2beb8-4b30-48c4-8ba6-5e8ab88ae456.png)
+![ATP Message Disposition Report data for a day](../../media/68d2beb8-4b30-48c4-8ba6-5e8ab88ae456.png) the aggregate view of the report only allows for 90 days filtering, while the detial view only allows for 10 days filtering.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/view-reports-for-atp.md]:
```suggestion
![ATP Message Disposition Report data for a day](../../media/68d2beb8-4b30-48c4-8ba6-5e8ab88ae456.png)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by mypil** [comment]
@faoquong - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**